### DISCUSSION TITLE
Добавление сторонних языков

### DISCUSSION THREAD

**Discussion by Kukushechka7-ezx** [Sokol-Games-GmbH/PixelCombats.PublicApi#124] [Ideas]
Сделать возможность писать скрипты на сторонних языках по типу C# - C++

**Comment by forwork10** [comment]
ты на js сначала научись...

**Comment by forwork10** [comment]
темболее, что это делает в issue

**Comment by kkohno** [comment]
это пока не планируется, однако C# рассматривается для серверов, но это врядли... есть ограничения
